docs/build-the-docs.md

```diff
@@ -25,17 +39,17 @@ There, you can see:
 - The file it was found in
 - The line of that file where it is located
 
-## Viewing Changes
+## Viewing Changes
 
-Assuming that there are no errors, the next thing to do is to run the NPM web server so that you can view your changes, before committing and pushing the changes to the remote docs repository.
+Assuming that there are no errors, the next thing to do is to run the [NPM Serve tool](https://www.npmjs.com/package/serve) (I suggest using it, as it’s trivial to start) so that you can view your changes, before committing and pushing the changes to the remote docs repository.
```

Comment:
``I suggest`` —> ``We suggest``
A short description about the benefit using it would be good.
For me the question came up as the headline above says „running antora from teh command line“ and then you refreence to an npm serve tool...